10up/classifai
# Update: Better Watson credentials settings

### Description of the Change

* Add inline help text about creating an account and a resource, e.g. under the header and before the fields:
* Expand the length of the URL and API key fields (large-text class)
* Hide the username field and just have API URL and API Key fields by default.
* Add a link toggle underneath saying something like `I have a username/password instead` that then shows a Username field and changes API Key to Password



### Alternate Designs
n/a


### Benefits
Better Watson credentials settings.


### Possible Drawbacks
n/a


### Verification Process

1. Go to admin > ClassifAI > Language Processing.
2. See the new description under 'Natural Language Understanding'.
3. See credentials section now has only 2 fields: 'API URL' and 'API Key'.
4. See the new toggle link say: 'Use a username/password instead?'.
5. Click that toggle link, a new filed 'API Username' appears and has value that is `apikey`.
6. Change value of API Username to something different from `apikey` and save settings.
7. See the toggle link after page load is 'Use an API Key instead?'.



### Checklist:






### Applicable Issues
Resolves #41
